src/daemon/gnet/pool.go

```diff
@@ -18,6 +18,8 @@ import (
 	"github.com/skycoin/skycoin/src/util/elapse"
 	"github.com/skycoin/skycoin/src/util/logging"
 	"github.com/skycoin/skycoin/src/util/utc"
+	"github.com/skycoin/skycoin/src/daemon/pex"
+	//"../pex"
```

Comment:
Remove comments

Reply:
These changes are irrelevant in this context . Please restart your PR using `develop` as base branch .